### ISSUE TITLE
Grant Suggestion: EU Next Generation Internet

### ISSUE THREAD

**Issue by tholop** [datproject/organization#7]
Hello,

The European Commission has been offering grants to fund innovative projects, notably related to decentralized web and privacy. They typically offer grants ranging from 50K to 200K euros for each project, each year.

I do not know whether Dat would be eligible, but in case you want to have a look here is a link:
[https://www.ngi.eu/news/2019/08/07/h2020-ict-calls-are-open-lets-build-the-internet-of-humans/](https://www.ngi.eu/news/2019/08/07/h2020-ict-calls-are-open-lets-build-the-internet-of-humans/)

By the way, I would be happy to help if I can!

Edit: I hope this is the right place to do such a suggestion, as I am a bit confused by the architecture of the community, repos, and multiple websites!

**Comment by okdistribute** [comment]
Yes thank you so much for the heads up, @tholop ! We should definitely try to apply. 

Edit: yes this is a great place, sorry it's confusing!
